### PR TITLE
fix https://github.com/AdguardTeam/AdguardFilters/issues/151369 video…

### PR DESCRIPTION
**Pull request by Yuki2718**
….unext.jp

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report
- [x] My code follows [syntax](https://adguard-dns.io/kb/general/dns-filtering-syntax/) of this project
- [x] I have performed a self-review of my own changes
- [x] My changes do not break web sites, apps and files structure

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads
- [x] Website or app doesn't work properly
- [ ] Missed analytics or tracker
- [ ] Filters maintenance

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/151369

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

1. Your comment

I managed to get an account and reproduced. https://github.com/AdguardTeam/AdguardFilters/commit/ef086b84bbac29e99dc6976403b09ade3f1f167e

2. Screenshots

<summary>Screenshot:</summary>

unext

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: Filters/exclusions.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/151369
+||beacon.unext.jp^
 ! totaladblock.com - is not used to show ads
```
Comment (DMW1971) [inline Filters/exclusions.txt:3]: 🔄👍

**Comment by Alex-302** [comment]
Only one site is affected?

**Comment by Yuki2718** [comment]
> Only one site is affected?

Look carefully, it's 1p tracker.

**Comment by Alex-302** [comment]
Removed from TPF.

**Pull request closed without merging**